databricks_cli/jobs/api.py

```diff
@@ -39,6 +39,11 @@ def delete_job(self, job_id):
     def get_job(self, job_id):
         return self.client.get_job(job_id)
 
+    def get_jobs_by_name(self, full_job_name):
+        jobs = self.client.list_jobs()['jobs']
+        result = list(filter(lambda job: job['settings']['name'] == full_job_name, jobs))
```

Comment:
What would happen if the jobName does not exist? or if there are more than one jobs with the same names?
Do we need unit tests for those behaviors?

Reply:
If jobName doesn't exist, it should be an empty list. I'll unit test it!